templates/customers/register.liquid

```diff
@@ -1,79 +1,73 @@
-{% layout settings.customer_layout %}
+{% layout 'bootstrap_migration' %}
 {% anti_forgery %}
-<main class="main" role="main">
-    <aside class="container">
-        <div ng-controller="accountRegisterController" ng-init="init('{{ shop.id }}')" ng-cloak>
-            <div ng-if="loader.isLoading" class="loader">
-                <div class="indicator"></div>
-            </div>
-            <div class="row">
-                <div class="col-md-8 col-md-offset-2 col-xs-12">
-                    <vc-errors message="errorMessage" errors="errors"></vc-errors>
-                    <wizard on-finish="finishedWizard()" indicators-position="top">
-                        <form name="step1Form" novalidate ng-class="{'ng-submitted': step1Form.$submitted}">
-                            <wz-step wz-title="{{ 'customer.register.step_personal' | t }}" canexit="step1Form.$valid">
-                                <h1>{{ 'customer.register.step_personal' | t }}</h1>
-                                <div class="form-group">
-                                    <label class="radio-inline">
-                                        <input ng-model="registration.type" value="Business" type="radio" ng-click="switchTemplate(registration.type)">
-                                        <span class="switch"></span>
-                                        <span class="name">Business Use</span>
-                                    </label>
-                                    <label class="radio-inline">
-                                        <input ng-model="registration.type" value="Personal" type="radio" ng-click="switchTemplate(registration.type)">
-                                        <span class="switch"></span>
-                                        <span class="name">Personal Use</span>
-                                    </label>
+<main class="container pt-4 pb-5 flex-grow-1" role="main">
+    <div class="row justify-content-md-center" ng-controller="accountRegisterController" ng-init="init('{{ shop.id }}')" ng-cloak>
+        <div ng-if="loader.isLoading" class="loader">
+            <div class="indicator"></div>
+        </div>
```

Comment:
May we move it to angular.js component instead and use standard bootstrap classes on that: `bg-dark position-fixed d-flex align-items-center justify-content center` classes. You will only need introduce custom `bg-semitransparent` modifier (see old scss) and `t-0` (looks good helper, to probably `l-0 r-0 b-0` should exist too).

Similar for indicator: just use font awesome icon and `text-white`

Reply:
done